tests/dummy/app/templates/getting-started/assets.hbs

```diff
@@ -1,4 +1,16 @@
 {{! template-lint-disable block-indentation }}
+<h1 id="included-assets">Included Assets</h1>
+
+<p>ember-bootstrap uses the Bootstrap CSS classes while replacing the behaviors from the components Bootstrap implements
+    in <code>bootstrap.js</code>, such as toggle, navbar, modal, etc., with equivalent, CSS class-compatible
+    equivalents.</p>
```

Comment:
Instead of "equivalent... equivalents" maybe "equivalent, CSS class-compatible native Ember components"